### PR TITLE
Organize the data in the detail page properly, add rough styling

### PR DESCRIPTION
**Pull request by rcoh**
Screenshot 2020-05-25 23 08 41

Note: things like long descriptions that would flow onto multiple lines are totally untested.

### CODE REVIEW

File: apps/reporting/templatetags/extras.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from datetime import date
+
+import humanize
+from django import template
+
+register = template.Library()
+
+
+def ago(value):
+    delta = humanize.naturaldelta(date.today() - value)
```
Comment (mjacksonw) [inline apps/reporting/templatetags/extras.py:10]: Is this for more control than provided by https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#timesince?

Comment (rcoh) [reply apps/reporting/templatetags/extras.py:10]: heh, no. I just didn't know about timesince

Comment (rcoh) [reply apps/reporting/templatetags/extras.py:10]: Although, this is a heinous behavior: ```Comparing offset-naive and offset-aware datetimes will return an empty string.```

Comment (rcoh) [reply apps/reporting/templatetags/extras.py:10]: sure. why not error, definitely what the user wanted was for their thing to just disappear

Comment (rcoh) [reply apps/reporting/templatetags/extras.py:10]: although it does make it easy to add `ago` in one place since we use it everywhere

Comment (mjacksonw) [reply apps/reporting/templatetags/extras.py:10]: Django template filters in general return an empty string if there's an error – reason being they figure a missing string is desirable to a 500.

**Pull request merged**